src/github.rs

```diff
@@ -0,0 +1,31 @@
+use hyper::Client;
+use hubcaps::{Github, ReleaseOptions};
+use error::Error;
+use super::USERAGENT;
+use config::Config;
+
+pub fn release(config: &Config, tag_name: &str, tag_message: &str) -> Result<(), Error> {
+    let user      = &config.user[..];
+    let repo_name = &config.repository_name[..];
+    let branch    = &config.branch[..];
+    let token     = config.gh_token.as_ref().unwrap();
+
+    let client = Client::new();
+    let github = Github::new(USERAGENT, &client, Some(&token[..]));
```

Comment:
I defined it in main: https://github.com/semantic-rs/semantic-rs/blob/github-release/src/main.rs#L37
```rust
const USERAGENT: &'static str = concat!("semantic-rs/", env!("CARGO_PKG_VERSION"));
```

It's passed as the HTTP useragent when sending requests to GitHub